packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/l10n/network_lookup/lib_pin_code_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nLibPinCodeLookup extends I18nLibPinCodeLookup {
           bundledI18n = I18nLibPinCode(I18nLibPinCodeLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nLibPinCode(I18nLibPinCodeLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong